src/styles/selectEvent.css

```diff
@@ -0,0 +1,44 @@
+.registration-bg {
```

Comment:
Instead of writing custom CSS, could we use Chakra UI's styling? Try not to introduce custom CSS files unless absolutely necessary. For example, background image/position/size, flex, and positioning can all be done with Chakra UI styling: https://v2.chakra-ui.com/docs/styled-system/style-props#color-and-background-color